backend/e4h-services/ingestion-service/app/utils/project_service_client.py

```
@@ -17,46 +20,50 @@ def update_facility_with_supervisor(self, facility_payload:Dict[str,Any]):
-            print(f"An error occurred: {req_err}")
+            logger.error(f"Request error updating facility with supervisor: {req_err}", exc_info=True)
             raise req_err
 
     def create_project(self, project_payload: Dict[str, Any]):
         url = f"{self.project_service_url}/project/v1/_create"
         headers = {
             "Content-Type": "application/json"
         }
+        logger.trace(f"Creating project: {url}")
         try:
             response = requests.post(url, headers=headers, json=project_payload)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)